sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -777,6 +780,31 @@ public sealed partial class TrueRuleFilter : Azure.Messaging.ServiceBus.Manageme
         public override string ToString() { throw null; }
     }
 }
+namespace Azure.Messaging.ServiceBus.Plugins
+{
+    public abstract partial class ServiceBusPlugin
+    {
+        protected ServiceBusPlugin() { }
+        public abstract string Name { get; }
+        public virtual bool ShouldContinueOnException { get { throw null; } }
```

Comment:
How does this flag work?

Reply:
Plugin author can try/catch those on their own, right?

Reply:
Yes, this was in Track 1. @SeanFeldman / @nemakam any feedback here?

Reply:
I would rather we don't have it. This kind of semantics are very rare in the framework.

Reply:
> Plugin author can try/catch those on their own, right?

This was never about what authors can or cannot do. Rather, what end-users want.
I do agree that if a plugin author decided to provide that kind of flexibility they should make the plugin configurable and end-users indicate to the plugin wherever it should throw or not on exceptions.

👍 to remove it.